lib/include/platform/atomic_user_override_template.h

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
This file provides a template for user. It will not compile, which is intentional.